.ci/publish-images.sh

```diff
@@ -6,7 +6,7 @@ OPERATOR_VERSION=${OPERATOR_VERSION:-$(git describe --tags)}
 ## if we are on a release tag, let's extract the version number
 ## the other possible value, currently, is 'master' (or another branch name)
 ## if we are not running in the CI, it fallsback to the `git describe` above
-if [[ $OPERATOR_VERSION == release* ]]; then
+if [[ $OPERATOR_VERSION == release* || $OPERATOR_VERSION == v* ]]; then
```

Comment:
If the build is only triggered once - wouldn't the `OPERATOR_VERSION` just be `release*` or `v*` - i.e. shouldn't we only need to check for one of these values?

Reply:
Sorry, should have been clearer: the `release/v1.2.3` tag is still executed on its own build. This is where the `release.sh` script is executed, and a new commit is generated (like f840dc49). This new commit is tagged as `v1.2.3` and is the one that was triggering two builds before, and only one build now.

Reply:
Should it be removed?